float64 radius	# radius of the cylinder object
float64[] direction_vec # Direction vector towards the z-axis of the cylinder.
float64[] center_pt # Center point of the cylinder.
float64 height # Height of the cylinder.
geometry_msgs/PoseStamped object_pose
geometry_msgs/PoseStamped[] grasping_poses

